repositories.go

```diff
@@ -0,0 +1,79 @@
+package gitquery
+
+import (
+	"io"
+
+	"gopkg.in/src-d/go-mysql-server.v0/sql"
+)
+
+type repositoriesTable struct {
+	pool *RepositoryPool
+}
+
+func newRepositoriesTable(pool *RepositoryPool) sql.Table {
+	return &repositoriesTable{pool: pool}
+}
+
+func (repositoriesTable) Resolved() bool {
+	return true
+}
+
+func (repositoriesTable) Name() string {
+	return repositoriesTableName
+}
+
+func (repositoriesTable) Schema() sql.Schema {
+	return sql.Schema{
+		{Name: "id", Type: sql.Text, Nullable: false},
+	}
+}
+
+func (r *repositoriesTable) TransformUp(f func(sql.Node) sql.Node) sql.Node {
+	return f(r)
+}
+
+func (r *repositoriesTable) TransformExpressionsUp(
+	f func(sql.Expression) sql.Expression) sql.Node {
+
+	return r
+}
+
+func (r repositoriesTable) RowIter() (sql.RowIter, error) {
+	iter := &repositoriesIter{}
+
+	rowRepoIter, err := NewRowRepoIter(r.pool, iter)
+	if err != nil {
+		return nil, err
+	}
+
+	return rowRepoIter, nil
+}
+
+func (repositoriesTable) Children() []sql.Node {
+	return []sql.Node{}
```

Comment:
can we return `nil` instead to save an alloc here?

Reply:
Done, we should also change this in all other tables.